okhttp-tests/src/test/java/okhttp3/internal/http2/HttpOverHttp2Test.java

```diff
@@ -86,28 +89,61 @@
 import static org.junit.Assert.fail;
 
 /** Test how SPDY interacts with HTTP/2 features. */
+@RunWith(Parameterized.class)
 public final class HttpOverHttp2Test {
   private static final Logger http2Logger = Logger.getLogger(Http2.class.getName());
+  private static final SslClient sslClient = SslClient.localhost();
+
+  @Parameters
```

Comment:
Can you change this so the IDE makes it clear what is running

`  @Parameters(name="{2}")`

![image](https://user-images.githubusercontent.com/231923/36639094-f336ef8c-19fb-11e8-8f44-ec134c5776ba.png)